dlhub_sdk/tests/dlhub_gha_test.py

```diff
@@ -0,0 +1,229 @@
+import os
+
+from pytest import fixture, raises, mark
+
+from dlhub_sdk.models.servables.python import PythonStaticMethodModel
+from dlhub_sdk.utils.futures import DLHubFuture
+from dlhub_sdk.client import DLHubClient
+from mdf_connect_client import MDFConnectClient
+import mdf_toolbox
+
+
+#github specific declarations
+client_id = os.getenv('CLIENT_ID')
+client_secret = os.getenv('CLIENT_SECRET')
+
+fx_scope = "https://auth.globus.org/scopes/facd7ccc-c5f4-42aa-916b-a0e270e2c2a9/all"
+
+services=[
+        "search",
+        "dlhub",
+        fx_scope,
+        "openid"]
+
+auth_res = mdf_toolbox.confidential_login(client_id=client_id,
+                                        client_secret=client_secret,
+                                        services=services, make_clients=True)
+
+# Check if we are on travis
+#  See: https://blog.travis-ci.com/august-2012-upcoming-ci-environment-updates
+is_travis = 'HAS_JOSH_K_SEAL_OF_APPROVAL' in os.environ

```

Comment:
We'll have to modify this logic to detect if we are on GitHub actions.

Reply:
logic modified -- but there were two tests that had a skipif tag with travis logic -- test_submit and test_namespace in dlhub_gha_test.py. I deleted the skipifs for now, and test_submit seemed to work, test_namespace failed.